CONTRIBUTING.md

```diff
@@ -0,0 +1,47 @@
+## Set up
```

Comment:
I think of `contributing.md` more as "how to do PRs" than "dev setup". Just the name of it might be slightly misleading - partners probably won't be planning on contributing any changes to the codebase.

Maybe setup should live in the README.md?

Reply:
related: do we want to encourage potential partners to run our code to get a feel for things?

If it's useful for them to run, we may suggest explicitly making a dev add-on to test this against, and explicitly for them NOT to base their production code off of a fork/clone of this (maybe in stronger language than we have now).

If it's not useful for them to run, we may explicitly suggest NOT running it, and encourage them to only look at this for a reference.